pom.xml

```diff
@@ -150,8 +150,7 @@
           -->
         <maven.min.version>3.6.0</maven.min.version>
         <!-- Require Java 11 -->
-        <maven.compiler.target>11</maven.compiler.target>
-        <maven.compiler.source>11</maven.compiler.source>
```

Comment:
I think we need to keep these. While they'll be ignored if `<release/>` is used, but we don't explicitly set that on the `maven-compiler-plugin`. However, we do explicitly set the `<target/>` and `<source/>` settings. By default too the `jdk.min.version` is set to `maven.compiler.argument.source` which is set from `maven.compiler.source`.